kms/api/quickstart.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+
+def run_quickstart():
+    # [START kms_quickstart]
+    # Imports the Google APIs client library
+    from googleapiclient import discovery
+    from oauth2client.client import GoogleCredentials
```

Comment:
This is no longer needed, googleapiclient can figure out application default credentials on its own.